community-membership.md

```diff
@@ -1,77 +1,198 @@
+- Added to [`kubernetes-reviewers`](https://github.com/orgs/kubernetes/teams/kubernetes-reviewers)
+- Granted "read access" to kubernetes repo
+- Can champion incubator repos
+- May get a badge on PR and issue comments
+
+## Approver
+
+Code approvers are able to both review and approve code contributions.  While code review is focused on
+code quality and correctness, approval is focused on holistic acceptance of a contribution including:
+backwards / forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues,
+interactions with other parts of the system, etc.
+
+**Defined by:** *approver* entry in an OWNERS file in the kubernetes codebase
+
+Approver status is scoped to a part of the codebase.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be an approver in an
+[OWNERS](contributors/devel/owners.md) file (for repos using the bot).
+
+- Reviewer of the codebase for at least 3 months
+- Primary reviewer for at least 10 substantial PRs to the codebase
+- Reviewed or merged at least 30 PRs to the codebase
+- Nominated by an area/component owner
+  - With no objections from other owners
+  - Done through PR to update the top-level OWNERS file
+
+### Responsibilities and privileges
+
+The following apply to the part of codebase for which one would be an approver in an
+[OWNERS](contributors/devel/owners.md) file (for repos using the bot).
+
+- Approver status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgement
+- Responsible for project quality control via [code reviews](contributors/devel/collab.md)
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations](contributors/devel/community-expectations.md);
+- Mentor contributors and reviewers
+- May approve code contributions for acceptance
+
+## Owner
+
+Owners of an area / component are approvers of an entire area that have demonstrated good judgement and
+responsibility.  Owners accept design proposals and approve design decisions for their area of ownership.
+
+**Defined by:** *approver* entry in the top-level OWNERS file for the area in the kubernetes codebase
+
+### Requirements
+
+The following apply to the area / component for which one would be an owner.
+
+- Originally authored or contributed major functionality to an area
+- An approver in the **top-level** [OWNERS](contributors/devel/owners.md) files
+
+### Responsibilities and privileges
+
+The following apply to the area / component for which one would be an owner.
+
+- Owner status may be a precondition to accepting a new component or piece of major functionality
+- Design/proposal approval authority over the area / component, though escalation to kubernetes-maintainers is possible.
```

Comment:
nit: line 100 uses single backticks around github team names, and links to the team. we should do that here too